numba/cuda/ufuncs.py

```diff
@@ -272,4 +272,55 @@ def np_real_atanh_impl(context, builder, sig, args):
 
     db[np.degrees] = db[np.rad2deg]
 
+    db[np.bitwise_and] = {
+        'll->l': numbers.int_and_impl,
```

Comment:
Why do we not have implementations for unsigned types?

Reply:
Thanks! I added the missing types from `np.<ufunc>.types` except `O` which is a generic python object